### PR TITLE
Update russian localisation

### PR DESCRIPTION
**Pull request by minegame2021**

### CODE REVIEW

File: src/public/assets/lang/ru.yaml [hunk 1]
```diff
@@ -31,88 +31,88 @@ docs: Доки
...
 with everything in one place, do anything anywhere at anytime.: Всё в одном месте, делай всё что угодно в любое время
 easily apply layout: Простое изменение расположения кнопок
-with a list of presets determined from you apps, easily get to work with the perfect window layout. not enough? change it in the settings.: С предустановками от твоих программ можно легко работать с идеальным расположением кнопок. Этого мало? Изменяй всё что угодно в настройках
-overpowered dock: Имбовый таскбар
-your dock can do more than ever. it's your ultimate manager to get you organised.: Таскбар может иметь больше фичей чем когда либо, он поможет тебе быть более организованным 
+with a list of presets determined from you apps, easily get to work with the perfect window layout. not enough? change it in the settings.: С предустановками от твоих программ можно легко работать с идеальным расположением кнопок. Этого мало? Изменяй всё что угодно в настройках.
+overpowered dock: Имбовая док-панель
+your dock can do more than ever. it's your ultimate manager to get you organised.: Док-панель может иметь больше фичей чем когда либо. Она поможет тебе быть более организованным .
```
Comment (LazyCat2) [inline src/public/assets/lang/ru.yaml:62]: please, fix spase before dot

File: src/public/assets/lang/ru.yaml [hunk 2]
```diff
@@ -31,88 +31,88 @@ docs: Доки
 demo: Демо
 
 # index.tsx
-your pc but even better!: Твой ПК, но ещё лучше
-try in your browser: Посмотри на это в браузере
+your pc but even better!: Ваш ПК, но ещё лучше.
```
Comment (xAvatar6969) [inline src/public/assets/lang/ru.yaml:34]: There should be a "!", not a "."

**Review by xAvatar6969 — CHANGES_REQUESTED** [review]

**Comment by AZProductions** [comment]
Any updates to this pr @minegame2021 ? Or should we close it?

**Comment by froxcey** [comment]
I will start reviewing after you address the changes requested, or just tell me if you think this is good already

**Comment by froxcey** [comment]
I guess I will edit this PR myself later if no one want to take care of this.

**Review by froxcey — APPROVED** [review]
I made the changes as requested. I will merge this tomorrow so people can have time to review.

**Review by AZProductions — COMMENTED** [review]
LGTM

**Comment by froxcey** [comment]
Alr merging now

**Pull request merged**